### DISCUSSION TITLE
img not working on next export anymore like before , since next-Images not supported with next export then let me use just img like before new nextjs

### DISCUSSION THREAD

**Discussion by ayoubwaza** [vercel/next.js#26409] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

14

### What browser are you using?

google chrome

### What operating system are you using?

w10

### How are you deploying your application?

next export

### Describe the Bug

img not working on next export anymore like before , since next-Images not supported with next export then let me use just img like before new nextjs

### Expected Behavior

img not working on next export anymore like before , since next-Images not supported with next export then let me use just img like before new nextjs

### To Reproduce

img not working on next export anymore like before , since next-Images not supported with next export then let me use just img like before new nextjs

**Comment by timneutkens** [comment]
Converted to discussions as the issue template was not followed, please fill out all sections correctly including providing a reproduction and clear description of what you'd like to change.
